packages/react/popover/src/Popover.tsx

```diff
@@ -224,7 +224,7 @@ const PopoverContentImpl = forwardRefWithAs<typeof PopperPrimitive.Root, Popover
       <PortalWrapper>
         <ScrollLockWrapper>
           <FocusScope
-            trapped={trapFocus}
+            trapped={skipCloseAutoFocus ? false : trapFocus}
```

Comment:
can we rename `skipCloseAutoFocus` to something like `isInteractOutsideClose` or `isCloseOnInteractOutside` 😅 🙏  ... 

I'm trying to name it something that reads like:

```jsx
trapped={if closing via an outside interaction then don't trap, otherwise trap}
````

Seems to fit `onCloseAutoFocus` too:


```jsx
onCloseAutoFocus={(event) => {
  if (isCloseOnInteractOutside) {
    event.preventDefault();
  } else {
    context.triggerRef.current?.focus();
  }
}}
```

I'm sure you can come up with something better but hopefully this explains what I'm trying to get at.